back/spec/acceptance/baskets_spec.rb

```diff
@@ -100,69 +122,17 @@
       let(:basket_id) { @basket.id }
 
       describe do
-        let(:idea_ids) { create_list(:idea, 3).map(&:id) }
+        let(:new_ideas) { create_list(:idea, 2, project: @project) }
+        let(:idea_ids) { new_ideas.map(&:id) + [@ideas.first.id] }
+        let(:baskets_ideas_attributes) { new_ideas.map { |idea| { idea_id: idea.id } } + [{ idea_id: @ideas.first.id, votes: 4 }] }
 
         example_request 'Update a basket' do
-          expect(response_status).to eq 200
+          assert_status 200
           json_response = json_parse(response_body)
           expect(json_response.dig(:data, :relationships, :ideas, :data).pluck(:id)).to match_array idea_ids
-        end
-      end
-
-      example "'baskets_count' is not updated when adding/removing the idea to an unsubmitted basket", document: false do
```

Comment:
I removed all these tests in my PR too!

Reply:
I removed them because I saw you removed them. So I don't waste time making them green :)